design-proposals/backend-storage-clone-on-migration/backend-storage-clone-on-migration.md

```diff
@@ -0,0 +1,92 @@
+# Overview
+This proposal is about supporting the migration of all VMs that use a persistent feature, such as persistent vTPM.
+
+To make this document as simple as possible, let's consider that storage in Kubernetes can either be filesystem (FS) or
+block mode, and either ReadWriteOnce (RWO) or ReadWriteMany (RWX) access mode, with storage classes supporting any
+number of combinations of both.
+
+As of KubeVirt 1.3, Here is how each of the 4 combinations is supported by the backend-storage for persistent VM state:
+- RWO FS: supported by backend-storage but makes VMs non-migratable. **This entire design proposal is about removing that limitation.**
+- RWX FS: preferred combination, supported by backend-storage and compatible with VM migrations
+- RWO block: not supported yet, but PRs exist to implement it in a non-migratable way 
+- RWX block: same as above, with migratability functional but considered unsafe, because it leads to 2 pods (source and target virt-launchers) mounting the same ext3 filesystem concurrently.
+
+The above is the best we can do with the current implementation, which involves mounting the same partition on migration source and target.  
```

Comment:
```suggestion
The above is the best we can do with the current libvirt implementation, which involves mounting the same partition on migration source and target.  
```

I'm not at all sure I understand to which "current implementation" you refer.

Reply:
I'll rephrase, what I mean is "without partitioning block / creating new PVCs on migration / libvirt block support"